tests/ToctreeBuilderTest.php

```diff
@@ -29,35 +31,43 @@ public function testBuildToctreeFiles(): void
 test1
 *
 test4
+banana
 EOF;
 
         $node->expects(self::once())
             ->method('getValueString')
             ->willReturn($toc);
 
-        $environment->expects(self::at(0))
-            ->method('absoluteUrl')
-            ->with('test1')
-            ->willReturn('/test1');
-
         $this->globSearcher->expects(self::once())
             ->method('globSearch')
             ->with($environment, '*')
-            ->willReturn(['/test1', '/test2', '/test3']);
+            ->willReturn(['/test1', '/test2', '/test3', '/apple', '/current_filename']);
+
+        $environment->expects(self::any())
```

Comment:
this kind of call doesn't actually do anything, let's remove it.
```suggestion
        $environment
```

Reply:
good tip!